Firmware/ultralcd.cpp

```diff
@@ -3472,8 +3472,21 @@ int8_t lcd_show_multiscreen_message_yes_no_and_wait_P(const char *msg, bool allo
 {
     return lcd_show_multiscreen_message_two_choices_and_wait_P(msg, allow_timeouting, default_yes, _T(MSG_YES), _T(MSG_NO));
 }
-//! @brief Show multiple screen message with two possible choices and wait with possible timeout
-//! @param msg Message to show
+//! @brief Show a two-choice prompt on the last line of the LCD
+//! @param first_selected Show first choice as selected if true, the second otherwise
+//! @param first_choice text caption of first possible choice
+//! @param second_choice text caption of second possible choice
+void lcd_show_two_choices_prompt_P(bool first_selected, const char *first_choice, const char *second_choice)
+{
+    lcd_set_cursor(0, 3);
+    lcd_print(first_selected? '>': ' ');
+    lcd_puts_P(first_choice);
+    lcd_set_cursor(7, 3);
```

Comment:
Can you change it to `lcd_set_cursor(10, 3);` so the `Yes/No` translations can both be up to 8 chars?

Reply:
We need to ensure that all the possible translation for "choice 2" (not just no - but all the wizard stuff) fits into 8 characters though.

Otherwise that's pretty trivial

Reply:
Done!